src/Forms/FormHandler.php

```diff
@@ -449,7 +449,7 @@ class FormHandler
     /** @var string */
     public string $FG_LIST_ADDING_BUTTON_MSG2;
 
-    public function __construct(string $tablename, string $instance_name, string $primary_key = "id")
+    public function __construct(string $tablename = "", string $instance_name = "", string $primary_key = "id")
```

Comment:
I don't have a full picture of what this change would do, I just fixed the invalid uses in a07c49f1 so this is no longer necessary